app.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const app = express()
+const getTopics  = require('./controllers.topic/topic.controller')
+app.use(express.json())
```

Comment:
line 4 isn't required at this point, there's no harm in it but it won't take affect until you get to a `POST` or `PATCH` endpoint

Reply:
do i need to remove this ?